models/groups.go

```diff
@@ -96,6 +97,22 @@ func (g *Group) BuildListUsersInGroupRequest(userPoolId string) *cognitoidentity
 	}
 }
 
+// BuildListUsersInGroupRequest builds a correctly populated ListUsersInGroupInput object with Next Token
+func (g *Group) BuildListUsersInGroupRequestWithNextToken(userPoolId string, nextToken string) *cognitoidentityprovider.ListUsersInGroupInput {
+	if nextToken == "" {
+		return &cognitoidentityprovider.ListUsersInGroupInput{
+			GroupName:  &g.Name,
+			UserPoolId: &userPoolId,
+		}
+	} else {
+		return &cognitoidentityprovider.ListUsersInGroupInput{
+			GroupName:  &g.Name,
+			UserPoolId: &userPoolId,
+			NextToken:  &nextToken,
+		}
+	}
```

Comment:
The `else` not required here as there is a return inside the `if` block. Again no functional difference its just less code/indentation which I think makes it easier to read.
```
if nextToken == "" {
    return &cognitoidentityprovider.ListUsersInGroupInput{
	GroupName:  &g.Name,
	UserPoolId: &userPoolId,
}

return &cognitoidentityprovider.ListUsersInGroupInput{
    GroupName:  &g.Name,
    UserPoolId: &userPoolId,
    NextToken:  &nextToken,
}
```

Reply:
Thank you